src/main/java/com/saiko/bidmarket/product/entity/Product.java

```diff
@@ -33,7 +33,7 @@
 
 @Entity
 @Getter
-@EqualsAndHashCode(of = { "id" }, callSuper = false)
+@EqualsAndHashCode(of = {"id"}, callSuper = false)
```

Comment:
엔티티에 해당 롬복 어노테이션을 사용하면 `Using @EqualsAndHashCode for JPA entities is not recommended. It can cause severe performance and memory consumption issues. ` 과 같은 경고가 뜨지 않나요?

Reply:
https://blog.gangnamunni.com/post/hibernate_lombok_lazy/

Reply:
해당 내용 관련 검색을 해봤는데 저희처럼 jpa와 @EqualsAndHashCode 를 사용할때 발생할 수 있는 여러 문제점 때문에 경고 로그를 띄우는 것 같습니다.
1. id의 변동 가능성이 있을때 문제점
2.toString() 사용시 프록시 객체를 초기화할 때 트랜잭션 밖에서 초기화하여 발생하게 되는 문제

관련 내용 노션페이지에 정리하여 두겠습니다. 나중에 같이 논의하여 수정하면 될 것 같습니다.